tests/include/tc_util.h

```diff
@@ -67,22 +67,19 @@
 	} while (0)
 
 #define TC_PRINT(fmt, ...) PRINT_DATA(fmt, ##__VA_ARGS__)
-#define TC_START(name) PRINT_DATA("tc_start() - %s\n", name)
+#define TC_START(name) PRINT_DATA("start() - %s\n", name)
```

Comment:
come to think about this, this will be also confusing, so what about just "starting test <testname>"?